spec/spec_helper.rb

```diff
@@ -37,6 +37,8 @@
     # `true` in RSpec 4.
     mocks.verify_partial_doubles = true
   end
+  config.filter_run :focus
+  config.run_all_when_everything_filtered = true
```

Comment:
Please don't change the spec_helper as part of this PR.  If this is a change that matters to you, please make a separate PR for it.